assignments/arrays.js

```diff
@@ -63,32 +63,56 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+// console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+let carModels = [];
+for (let i = 0; i < inventory.length; i++)
+    carModels.push(inventory[i].car_model);
+console.log(carModels.sort());
+
+let carMakes = [];
+for (let i = 0; i < inventory.length; i++)
+    carMakes.push(inventory[i].car_make);
+console.log(carMakes.sort());
+
+let carYears = [];
+for (let i = 0; i < inventory.length; i++)
+    carYears.push(inventory[i].car_year);
+console.log(carYears.sort());
+
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}`);
+
+
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+
+
+// console.log();
+const lastCar = 0;
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
-let carModels = [];
-console.log();
+let carModel = [];
+for (let i = 0; i < inventory.length; i++)
+    carModel.push(inventory[i].car_model);
+console.log(carModel.sort());
 
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
```

Comment:
you do not have to use sort, you could have done something like this
let carYears = [];
for (let i = 0; i < inventory.length; i++) {
  carYears.push(inventory[i].car_year);
}
console.log(carYears);